internal/pkg/cli/pipeline_status.go

```diff
@@ -0,0 +1,73 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	"github.com/spf13/cobra"
+)
+
+type pipelineStatusVars struct {
+	*GlobalOpts
+	shouldOutputJSON bool
+	appName       string
+	pipelineName  string
+}
+
+type pipelineStatusOpts struct {
+	pipelineStatusVars
+}
+
+func newPipelineStatusOpts(vars pipelineStatusVars) (*pipelineStatusOpts, error) {
+	return &pipelineStatusOpts{
+		pipelineStatusVars: vars,
+	}, nil
+}
+
+// Validate returns an error if the values provided by the user are invalid.
+func (o *pipelineStatusOpts) Validate() error {
+	return nil
+}
+
+// Ask prompts for fields that are required but not passed in.
+func (o *pipelineStatusOpts) Ask() error {
+	return nil
+}
+
+// Execute displays the status of the pipeline.
+func (o *pipelineStatusOpts) Execute() error {
+	return nil
+}
+
+// BuildPipelineStatusCmd builds the command for showing the status of a deployed pipeline.
+func BuildPipelineStatusCmd() *cobra.Command {
+	vars := pipelineStatusVars{
+		GlobalOpts: NewGlobalOpts(),
+	}
+	cmd := &cobra.Command{
+		Hidden: true, // TODO: remove when ready for production!
+		Use:    "status",
+		Short:  "Show the status of your application's pipeline.",
+		Long:   "Show the status of each deployed pipeline's stages.",
+
+		Example: `
+Shows status of the deployed pipeline "pipeline-myapp-mycompany-myrepo".
+/code $ copilot pipeline status -n pipeline-myapp-mycompany-myrepo`,
```

Comment:
I made some comments on this wording in my other PR - maybe you can make the changes in that PR.